<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>582</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>582</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Openblocks</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="editor.qrc">:/placeholder-logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="titleString">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Openblocks Engine</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Copright 2025 GNU Lesser General Public License v2.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="viewLicense">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;/license&quot;&gt;View license...&lt;/a&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionString">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>unknown</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AboutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
